src/common/CMakeLists.txt

```diff
@@ -30,7 +30,11 @@ endif()
 if(${OQS_USE_SHA2_OPENSSL})
     set(SHA2_IMPL sha2/sha2_ossl.c)
 else()
-    set(SHA2_IMPL sha2/sha2_c.c)
+    set(SHA2_IMPL sha2/sha2.c sha2/sha2_c.c)
+    if (OQS_DIST_ARM64v8_BUILD OR OQS_USE_ARM_SHA2_INSTRUCTIONS)
+      set(SHA2_IMPL ${SHA2_IMPL} sha2/sha2_ni.c)
+      # set_source_files_properties(sha2/sha2_ni.c PROPERTIES COMPILE_FLAGS -march=armv8-a+crypto)
```

Comment:
Is this commented-out line required?

Reply:
Looks like in some cases it is, so I've updated this to be more flexible.